### DISCUSSION TITLE
Have an app idea and was wondering if i could use this library for it?

### DISCUSSION THREAD

**Discussion by jawadkalia** [fzyzcjy/flutter_rust_bridge#621] [Q&A] [answered]
Hello!

Thank you for writing this library.

The app idea is an Azure Service Bus GUI interface. There are many out there but i wanted to try it out with flutter and rust.
Flutter/Dart pub.dev did not really had many libraries that provide the amqp protocol 1.0.0 interface. Rust has a Crate called Dove which does provide that.

Therefore i would like the "frontend" to be flutter, and the "backend" to be rust.

With that context i think this library is perfect. but i am an amateur.

1. can i use any crate in the rust side? for example dove
2. can i implement the io and storage on both sides?
3. any architectural tips? or tips in general?

**Comment by fzyzcjy** [comment, marked as answer]
You are welcome!

1. Sure
2. Sure
3. Maybe read the book http://cjycode.com/flutter_rust_bridge/ :)

**Reply by jawadkalia** [reply]
Thank you!

**Reply by fzyzcjy** [reply]
You are welcome :)
